modules/httpcache/httpcache.go

```diff
@@ -17,16 +17,21 @@ import (
 )
 
 // AddCacheControlToHeader adds suitable cache-control headers to response
-func AddCacheControlToHeader(h http.Header, d time.Duration) {
+func AddCacheControlToHeader(h http.Header, d time.Duration, additionalDirectives []string) {
+	directives := make([]string, 0, 2+len(additionalDirectives))
```

Comment:
The non-prod codepath will only add 1 directive, is it ok to allocate 2?

Reply:
Apparently it's okay as `strings.Join` ignores nil elements.

Reply:
`make( type, len, cap)`, the 3rd argument is for capacity, the length of `directives` is still 0 at the beginning.

ps: not related to `strings.Join` 😁

Reply:
I see.